src/redux/actions/dictionaries/dictionaryActionCreators.js

```diff
@@ -137,7 +138,8 @@ export const fetchDictionaryConcepts = data => (dispatch) => {
       dispatch(isFetching(false));
       })
       .catch((error) => {
-        dispatch(isErrored(error.response.data));
+        error.response? dispatch(isErrored(error.response.data)):
```

Comment:
@collinewait, please add a space before the `?` logic operator.

Reply:
@abulojoshua1,  both issues have been fixed.